team.go

```diff
@@ -73,6 +83,20 @@ func teamRequest(path string, values url.Values, debug bool) (*TeamResponse, err
 	return response, nil
 }
 
+func billableInfoRequest(path string, values url.Values, debug bool) (*BillableInfoResponse, error) {
+	response := &BillableInfoResponse{}
+	err := post(path, values, response, debug)
+	if err != nil {
+		return nil, err
+	}
+
+	if !response.Ok {
+		return nil, errors.New(response.Error)
+	}
+
+	return response, nil
```

Comment:
You could do `return response.BillableInfo, nil` here and adjust per below comments.